proto/pdpb.proto

```diff
@@ -103,8 +103,9 @@ service PD {
 }
 
 message WatchGlobalConfigRequest {
-    string config_path = 1;
-    int64 revision = 2;
+    string source = 1;
```

Comment:
this is change will break the backward compatibility

Reply:
It is changed by accident.

Reply:
fixed